pkg/pluginmanager/manager.go

```diff
@@ -788,6 +764,7 @@ func installOrUpgradePlugin(p *discovery.Discovered, version string, installTest
 			spinner.SetFinalText(errMsg, log.LogTypeERROR)
 		}
 	}
+	spinner.SetFinalText(installedMsg, log.LogTypeINFO)
```

Comment:
May be we can optimize even further here:
```
	pluginErr := verifyInstallAndInitializePlugin(plugin, p, version, installTestPlugin)
	if pluginErr == nil && spinner != nil {
        	spinner.SetFinalText(installedMsg, log.LogTypeINFO)
	}
```

Reply:
Updated.